tests/4.5/target_teams_distribute/test_target_teams_distribute_map.F90

```diff
@@ -0,0 +1,135 @@
+!===--- test_target_teams_distribute_map.F90--------------------------------===//
+!
+! OpenMP API Version 4.5 Nov 2015
+!
+! This test uses the map clause on a target teams distribute clause to test that
+! the indicated data is maped on the target in the fashion that is indicated
+! with the map-type.  Each map-type that is valid in this situation (to, from,
+! alloc, and tofrom) are each used and tested.
+!
+!//===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_target_teams_distribute_map
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
+  OMPVV_TEST_OFFLOADING
+  OMPVV_TEST_SHARED_ENVIRONMENT
+  OMPVV_TEST_VERBOSE(test_map_to() + test_map_from() + test_map_alloc() + test_map_tofrom() .ne. 0)
```

Comment:
It might be better to keep these separate, so the VERBOSE will mention which one was the one that actually failed?

Reply:
Should be separate now